irods/test/zone_test.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+from __future__ import absolute_import
+import os
+import sys
+import unittest
+
+from irods.models import User,Collection
+from irods.zone import iRODSZone
+from irods.access import iRODSAccess
+from irods.collection import iRODSCollection
+from irods.exception import CollectionDoesNotExist
+import irods.test.helpers as helpers
+
+class TestRemoteZone(unittest.TestCase):
+
+    def setUp(self):
+        self.sess = helpers.make_session()
+
+    def tearDown(self):
+        '''Close connections
+        '''
+        self.sess.cleanup()
+
+    # This test should pass whether or not federation is configured:
+    def test_create_other_zone_user_227_228(self):
+        usercolls = []
+        session = self.sess
+        A_ZONE_NAME = 'twigliteZone'
+        A_ZONE_USER = 'rodsterling'
```

Comment:
can we stick with alice and bobby?   tempZone and otherZone?

Reply:
yes, no problem